### DISCUSSION TITLE
Populating version of the generated .exe

### DISCUSSION THREAD

**Discussion by seidhkona** [coreybutler/node-windows#319] [Q&A]
I wonder if there's a way to generate service .exe with a specific version (provided)? Currently, I see that the generated exe is using winsw.exe version.

**Comment by coreybutler** [comment]
I'm not clear what is being asked. Are you saying you want your application to show a different version number? OR do you want to use a different version of Node.js? Or do you want to use a different version of winsw.exe?

**Reply by seidhkona** [reply]
Sorry for being so unclear.. The more I think of it the more I realize that whatever I'm asking is most likely not really possible. If I understand correctly node-windows generates the service that points to an executable in daemon folder. That executable is essentially node windows service with a name of my service. If you look at the Details tab of the executable it will show description and version of winsw.exe. That's the version I was wondering about.

**Reply by coreybutler** [reply]
Ah, I see. There's no way to change that with winsw.exe, but I'll keep this in mind. I'm in the planning stages of for a new executable (i.e. replacement for winsw.exe). This seems like something that could be easily added.

**Comment by MarkAurit** [comment]
> The more I think of it the more I realize that whatever I'm asking is
most likely not really possible

Been there, done that

On Sat, Sep 3, 2022 at 12:02 AM seidhkona ***@***.***> wrote:

> Sorry for being so unclear.. The more I think of it the more I realize
> that whatever I'm asking is most likely not really possible. If I
> understand correctly node-windows generates the service that points to an
> executable in daemon folder. That executable is essentially node windows
> service with a name of my service. If you look at the Details tab of the
> executable it will show description and version of winsw.exe. That's the
> version I was wondering about.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/coreybutler/node-windows/discussions/319#discussioncomment-3538706>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAL2NV7HW3HPNBEYXSADQO3V4IXIVANCNFSM6AAAAAAQCR7ZNU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> .com>
>


-- 

=======================
***@***.***
